app/controllers/summoners_controller.rb

```diff
@@ -11,19 +11,23 @@ def new
   def create
     name = summoner_params[:name]
     summoner_info = Riot.summoner_name(name)[name]
-    params = {"riot_id" => summoner_info['id'], "name" => summoner_info['name']}
-    @summoner = Summoner.new(params)
-    if @summoner.save
-      flash[:notice] = 'summoner added'
-      teams_array = Riot.team(params["riot_id"])[@summoner.riot_id.to_s]
-      teams_array.each do |team|
-        Team.make(team, @summoner)
-      end
-      redirect_to summoner_path(@summoner)
+    params = {"riot_id" => summoner_info['id'], "name" => summoner_info["name"]}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.